service-pdf/tests/OpgTest/Lpa/Pdf/AbstractPdfTestClass.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+namespace OpgTest\Lpa\Pdf;
+
+use Opg\Lpa\Pdf\AbstractIndividualPdf;
+use Opg\Lpa\DataModel\Lpa\Lpa;
+use ConfigSetUp;
+use Opg\Lpa\Pdf\Aggregator\AbstractAggregator;
+use Opg\Lpa\Pdf\Config\Config;
+use Opg\Lpa\Pdf\PdftkFactory;
+use PHPUnit\Framework\TestCase;
+
+abstract class AbstractPdfTestClass extends TestCase
```

Comment:
Should this class be named AbstractPdfTestCase rather than Class?  I am always wary of a class name that has the word class in it.... ;)